examples/plotting/wcsaxes_plotting_example.py

```diff
@@ -0,0 +1,54 @@
+"""
+===========================
+WCS Axes Plotting
+===========================
+In this example we are going to look at how pixel and map coodinates work with WCS Axes.
+"""
+
+import matplotlib.pyplot as plt
+
+from astropy import units as u
+
+import sunpy.data.sample
+import sunpy.map
+
+###############################################################################
+# Lets start by creating a Map from the sample data.
+my_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+###############################################################################
+# Lets choose xlimits and ylimits in arcsecs that we are interested in.
+xlims_world = [500, 1100]*u.arcsec
+ylims_world = [-800, 0]*u.arcsec
+
```

Comment:
```suggestion
```

Reply:
these aren't used in this example.

Reply:
yeah totally. I added xlim and ylim as it was mentioned in the issue but couldn't use it later. will remove it for sure.